src/views/pages/globalping.html

```diff
@@ -3135,7 +3174,9 @@
 
 				if (lines[i].includes('---')) { break; }
 
-				packetLines.push(lines[i]);
+				let numberedLine = lines[i].replace(/icmp_seq=\d+/, `icmp_seq=${idxShift + i}`);
```

Comment:
I think this won't work correctly for this case:
```
no answer yet for icmp_seq=1
64 bytes from api.globalping.io (49.12.17.170): icmp_seq=1 ttl=51 time=15.9 ms
no answer yet for icmp_seq=2
no answer yet for icmp_seq=3
64 bytes from api.globalping.io (49.12.17.170): icmp_seq=3 ttl=51 time=15.3 ms
no answer yet for icmp_seq=4
```

Sum the shift with the actual number from the regex match because the numbers can repeat.